### ISSUE TITLE
Focus Appearance (Minimum) Understanding document challenges

### ISSUE THREAD

**Issue by mbgower** [w3c/wcag#1930]
This issue catches several considerations for the Understanding document that were too dramatic for a PR without an explanation of the changes, especially since they involve matters of interpretation. Covered here are:
- typical focus indicators
- wording of a note
- Figure 9 and its caption
- Figure 11 removal
- Addition of figure for 3D-drop shadow

### Typical focus indicators
In the 4th bullet of the Typical focus indicators list ( ⅓ through Understanding), it addresses insertion points. It got me wondering about something- whether we should have language about text editors (think of code editors) where the insertion point is the ONLY form of focus indicator. Maybe such an interface isn’t a UIC? Should that be noted?

### Wording of note
I have some problems with the following note:
> “If the control has a non-rectangular shape, the secondary size measure in the success criterion is not valid, the 1 CSS pixel perimeter is best suited for this purpose.”

With the addition of the bounding box definition, I think either area calculation still works. For example, the star’s bounding side is 136 and 4 times that is 544 CSS pixels. Concentrated into a minimum 2 CSS pixel thick area, that seems sufficient. I recommend this note be removed.

### Figure 9 and its caption
Figure 9 and its caption contradict the bounding box definition. It states: 
> Figure 9 The wrapping does not affect the calculation, it is considered one element for the purpose of calculating the focus indicator area.

This contradicts the definition of bounding box, which states:
> the smallest enclosing box within which all the points of a shape lie. Where a component consists of disconnected parts, such as a link that wraps onto multiple lines, each part is considered to have its own bounding box.

Thinking it over I think both the bounding box definition and the example need tweaking. First, on both Safari and Chrome, each of the link parts indicators are complete focus rectangles when broken across 2 lines. 
![Screen Shot 2021-06-07 at 3 29 18 PM](https://user-images.githubusercontent.com/14298245/123143288-414b8b80-d40f-11eb-942d-846d1e82cb95.png)
Safari focus rectangle on two-line link
<img width="905" alt="Screen Shot 2021-06-07 at 3 30 24 PM" src="https://user-images.githubusercontent.com/14298245/123143306-46a8d600-d40f-11eb-9906-2425ea6e3f4e.png">
Chrome focus rectangle on two-line link

Compare this to figure 9, which doesn’t close off the box.
![Screen Shot 2021-06-07 at 3 46 00 PM](https://user-images.githubusercontent.com/14298245/123143543-9091bc00-d40f-11eb-86b4-333b288cfd29.png)

However, when the link overlaps across two lines, the focus indicator for my browsers IS contiguous. Our wording does not seem to entirely cover these three variations in 2-line link focus indicators.
![Screen Shot 2021-06-07 at 3 36 12 PM](https://user-images.githubusercontent.com/14298245/123143454-79eb6500-d40f-11eb-965b-e64b17cf0671.png)

### Remove Figure 11
![image](https://user-images.githubusercontent.com/14298245/123144079-24638800-d410-11eb-9c3e-90b2d0a4577c.png)

I find that Figure 11 distracts from the flow of the document, and is more about the following section Adjacent contrast. i recommend it be removed.

### 3-D drop shadow
The final paragraph before the Not fully obscured section, covering 3D-drop shadow, could use a figure.

**Comment by alastc** [comment]
> Typical focus indicators

Interesting point, the scenario in mind was a small text-input, rather than a large text-editor area. What matters then is that you can tell which input is focused. For a larger WYSIWYG editor, the whole thing is pretty big and doesn't necessarily need a focus indicator around it, but the editing point is important.

 > Wording of note

It is worded a bit too emphatically, and I think it was from before we updated the SC text with the latest outline/shape bullets.

> Figure 9 and its caption
> on both Safari and Chrome, each of the link parts indicators are complete focus rectangles when broken across 2 lines.

True for outlines, but if you use a border it does break in the way shown in the figure, so there are at least 3 variations of this link type.

> Remove Figure 11

This is the primary example that shows why adjacent contrast in itself cannot work for this SC. That section has:

- Passing outline
- Failing outline
- Failing background change
- Passing background change.

If anything, perhaps we should change the order of the last two?

 > 3-D drop shadow
> The final paragraph before the Not fully obscured section, covering 3D-drop shadow, could use a figure.

Figures 7 & 8 show that type of thing, but thinking about it, now that we have a better scoping on the "Contrasting area", do we need that paragraph?

**Comment by alastc** [comment]
I've been working through a few issues with PR #2000, so from here I've:

- Removed the note that started "If the control has a non-rectangular shape..."
- Removed fig 11 ("The second link has a light-grey...")
- Added a figure for the 3D drop-shadow concept. It doesn't use a drop-shadow, but I think the concept is clear. (Anyone with better photoshop type skills can provide a replacement!)

<h3>Typical focus indicators</h3>

I think we do need to scope-out graphical text-editors from this, a focus indicator around that wouldn't be helpful in most situations. 

<h3>Figure 9 and its caption</h3>

I agree that using border (rather than outline) is unusual, I just want to have a separate issue to deal with #1328 before removing that, as we had agreed to include that.

**Comment by alastc** [comment]
Hi @mbgower, I think this issue has been dealt with #2000 and the separated issues. Closing now, but open if you disagree.
